DiscordBot.ActivityLogging/Helpers/LogSenderHelper.cs

```diff
@@ -0,0 +1,128 @@
+        /// </summary>
+        /// <param name="textChannel">Discord text channel that message will be send</param>
+        /// <param name="mention">User mention associated with message</param>
+        /// <param name="action">What text activity was done in text form</param>
+        /// <param name="title">Title of log message</param>
+        /// <returns>Task for discord SendMessageAsync()</returns>
+        private async Task SendTextChannelActivity(SocketTextChannel textChannel, string mention, string action, string title)
+        {
+            var time = GetTime(textChannel.Guild.Id);
+            var embed = BuildEmbed(title, $"[ {time} ] \r\n" + action);
+            await textChannel.SendMessageAsync(embed: embed);
+        }
+
+        /// <summary>
+        /// Method for sending formatted information about voice activity.
+        /// </summary>
+        /// <param name="textChannel">Discord text channel that message will be send</param>
+        /// <param name="mention">User mention associated with message</param>
+        /// <param name="action">What text activity was done in text form</param>
+        /// <returns>Task for discord SendMessageAsync()</returns>
+        private async Task SendVoiceChannelActivity(SocketTextChannel textChannel, string mention, string action)
+        {
+            var embed = BuildEmbed("Voice status change", $"[ {GetTime(textChannel.Guild.Id)} ]" + mention + " " + action);
+
+            await textChannel.SendMessageAsync(embed: embed);
+        }
+
+        /// <summary>
+        /// Method for obtaning time for specific guildId.
+        /// </summary>
+        /// <param name="guildId">Discord unique guild id</param>
+        /// <returns>Formatted current time in guild timezone</returns>
+        private string GetTime(ulong guildId)
+        {
+            var timezoneString = _configProvider.GetParameter(guildId, "TimeZoneValue");
+            var timezone = _timeZoneHelper.ConvertTimeZoneFromString(timezoneString);
+            var time = _timeZoneHelper.GetCurrentTimeZoneTime(timezone);
+            return time;
+        }
+
+        /// <summary>
+        /// Obtains guilds log channel based on database value.
+        /// </summary>
+        /// <param name="guildId">Discord unique guild id</param>
+        /// <returns>Text channel as Task</returns>
+        private async Task<SocketTextChannel?> GetLoggingTextChannel(ulong guildId)
+        {
+            var textChannelId = _configProvider.GetParameter(guildId, "LoggingChannelId");
+            return textChannelId is null ? null : await _client.GetChannelAsync(Convert.ToUInt64(textChannelId)) as SocketTextChannel;
+        }
+
+        /// <summary>
+        /// Method for building discord embed message with common formatting for logs.
+        /// </summary>
+        /// <param name="title"></param>
+        /// <param name="description"></param>
+        /// <returns></returns>
```

Comment:
I mean I get that the parameters' names are self descriptive, but don't make an empty dosc 😆